apex/contrib/openfold_triton/fused_adam_swa.py

```diff
@@ -14,27 +15,39 @@
 
 # The most common parameter size in open-fold.
 CHUNK_SIZE = torch.tensor(128, dtype=torch.int64)
+
+
 # Data type enumerates. tl.constexpr arg doesn't accept Triton data types.
-kFP16 = 0
-kBF16 = 1
-kFP32 = 2
-kFP64 = 3
+@unique
```

Comment:
Didn't know there exists this decorator

Reply:
Yeah, this is [a neat decorator](https://docs.python.org/3/library/enum.html#enum.unique) ensuring unique enum values ;)